packages/engine/bin/hash_engine/src/main.rs

```diff
@@ -8,8 +10,19 @@ use hash_engine_lib::{
     utils::init_logger,
 };
 
+#[derive(Debug)]
+pub struct EngineError;
+
+impl fmt::Display for EngineError {
+    fn fmt(&self, fmt: &mut fmt::Formatter<'_>) -> fmt::Result {
+        fmt.write_str("Could not run engine")
```

Comment:
While we're here can we change this too. If I'm right, this error could appear at any time during the Engine's execution, whereas the phrasing here _feels_ like we're saying the Engine couldn't start.

We can copy the same phrasing as from the CLI error.